packages/core/src/index.d.ts

```diff
@@ -0,0 +1,96 @@
+export interface TemplateOptions extends SvgrOpts {}
+
+export interface TemplateData {
+  imports?: string[]
+  interfaces?: string[]
+  componentName?: string
+  props?: string[]
+  jsx?: string
+  exports?: string[]
+}
+
+export type TemplateFunc = (
+  templateOptions: { template: unknown },
+  opts: TemplateOptions,
+  data: TemplateData,
+) => string
+
+export interface SvgrOpts {
+  /** Specify a custom config file. */
+  configFile?: string
```

Comment:
I think this object can also have the `plugins` attribute

```
/** Specify custom plugins installed. */
plugins?: string[];
```

https://github.com/gregberge/svgr/blob/f18ea80dda0eeda8863d44875a435892a36e8705/packages/core/src/__snapshots__/config.test.js.snap#L13
<img width="733" alt="Screen Shot 2021-09-16 at 21 36 11" src="https://user-images.githubusercontent.com/24610813/133705884-b37d24f2-7bdc-4b8a-838f-a23ea82f358b.png">